src/Umbraco.Cms.Api.Management/ViewModels/Document/DocumentVersionItemResponseModel.cs

```diff
@@ -0,0 +1,41 @@
+namespace Umbraco.Cms.Api.Management.ViewModels.Document;
+
+public class DocumentVersionItemResponseModel
+{
+    public DocumentVersionItemResponseModel(
```

Comment:
Do we want to do a mapping from ContentVersionMeta instead of that many params?

Reply:
If we were using it in more than one place yes, but this is fine, no need to introduce another layer.